### ISSUE TITLE
Sleuthkit build fails with current libewf-experimental

### ISSUE THREAD

**Issue by kamwoods** [sleuthkit/sleuthkit#642]
After successfully compiling the current libewf-experimental source in Ubuntu 16.04 with GCC 5.3.1, the sleuthkit build (from GitHub master) fails out as follows:

/tmp/sleuthkit/tsk/img/ewf.c:67: undefined reference to `libewf_handle_read_random'

This is because "libewf_handle_read_random" (itself a v2 libewf API patch vs "libewf_read_random") no longer exists in libewf (libewf_handle_read_buffer_at_offset may be an appropriate alternative).

I understand that TSK only "officially" supports a specific 2013 build of libewf, but that build is increasingly difficult to find, and the link at http://www.sleuthkit.org/sleuthkit/download.php actually goes to a SourceForge site that has been more or less abandoned (it now forwards the user to the GitHub, which includes only libewf-experimental builds as releases, and some 2014 builds via Google).

It would be nice to have ewf.c updated to accommodate the current libewf conventions.

**Comment by SuperGod** [comment]
change all libewf_handle_read_random in tsk/img/ewf.c to libewf_handle_read_buffer_at_offset may be ok

**Comment by chenhang0905** [comment]
I follow “INSTALL.txt” to compile the project , but I get some error.can you tell me how to compile  Sleuthkit?

**Comment by joachimmetz** [comment]
Blocked on https://github.com/sleuthkit/sleuthkit/pull/941

**Comment by blshkv** [comment]
would be good to see some progress here

**Comment by joachimmetz** [comment]
@blshkv there are tons of more critical issues reported [here](https://github.com/sleuthkit/sleuthkit/issues) I recommend this project to focus their immediate attention there.

Or better yet, maybe you, can help out to fix them.

**Comment by blshkv** [comment]
@joachimmetz I understand the luck of resources and this is totally fine. However, there are three open pull requests at least, and they are open for more than a year. I'm not sure why you are asking for help but not really accepting it

**Comment by uckelman-sf** [comment]
@simsong This issue was fixed by the combination of some long-ago PR and PR #2996, and should be closed.
